sklearn/model_selection/tests/test_validation.py

```diff
@@ -476,21 +478,27 @@ def split(self, X, y=None, labels=None):
 
 
 def test_cross_val_predict_input_types():
-    clf = Ridge()
+    iris = load_iris()
+    X, y = iris.data, iris.target
+    X_sparse = coo_matrix(X)
+    multioutput_y = np.column_stack([y, y[::-1]])
+
+    clf = Ridge(random_state=0)
```

Comment:
If you want to avoid 'sag' solver, which is not very efficient for very small datasets, you can use `fit_intercept=False`, and the 'auto' solver will choose 'sparse_cg'.